test/e2e/util.go

```diff
@@ -497,6 +497,33 @@ func waitForPodsRunningReady(ns string, minPods int, timeout time.Duration) erro
 	return nil
 }
 
+func logFailedContainers(ns string) {
+	c, err := loadClient()
+	if err != nil {
+		Logf("Failed to load client")
+		return
+	}
+	podList, err := c.Pods(ns).List(api.ListOptions{})
+	if err != nil {
+		Logf("Error getting pods in namespace '%s': %v", ns, err)
+		return
+	}
+	for _, pod := range podList.Items {
+		if res, err := podRunningReady(&pod); res && err == nil {
+			Logf("Ignoring Ready pod %v/%v", pod.Namespace, pod.Name)
```

Comment:
This will produce a lot of spam, especially in enormous cluster running 30k pods. I'm not against it, but only if it's really needed.